Controllers/PostTodoController.cs

```diff
@@ -27,8 +28,8 @@ public PostTodoController(ITodoService todoService, IUserService userService, IM
             this.mapper = mapper;
         }
 
-        [HttpPost]
-        [Authorize("Aserto")]
+        [HttpPost]        
+        [Check(objectID: "resource-creators", objectType: "resource-creator", relationName: "member")]
```

Comment:
I would rename `relationName` to `relation` for consistency with the arguments of the `Check` API (objectType, ObjectID, relation, subjectType, subjectID).